_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-external-media.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * REST API endpoint for the External Media connections.
+ *
+ * @package Jetpack
+ * @since 8.5.0
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * External Medie connections helper API.
+ *
+ * @since 8.5
+ */
+class WPCOM_REST_API_V2_Endpoint_External_Media extends WP_REST_Controller {
+
+	private static $services_regex = '(?P<service>google_photos|pexels)';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'external-media';
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/list/' . self::$services_regex,
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_external_media' ),
+				'permission_callback' => array( $this, 'permission_callback' ),
+				'args' => array(
+					'search' => array(
+						'description' => __( 'Media collection search term' ),
```

Comment:
You'll want to add the Jetpack textdomain here.

```suggestion
						'description' => __( 'Media collection search term', 'jetpack' ),
```